lib/utilities/strategy-helpers/deployment-helper.js

```diff
@@ -157,7 +157,7 @@ function labelResources(files, kubectl, label) {
     let workflowName = process.env.GITHUB_WORKFLOW;
     workflowName = workflowName.startsWith('.github/workflows/') ?
         workflowName.replace(".github/workflows/", "") : workflowName;
-    const labels = [`workflowFriendlyName=${workflowName}`, `workflow=${label}`];
+    const labels = [`workflowFriendlyName=${workflowName.replace(/ /g, "_")}`, `workflow=${label}`];
```

Comment:
What if there were trailing spaces? Like `New   Workflow.yml`; then it would be `New___Workflow.yml` right? Could you fix that to just replace it to `New_Workflow.yml`

Reply:
I am kind of okay with multiple underscores. (Retains the original text to some extent and avoids unnecessary complexity).

Reply:
Not sure if we should do this. What should be output if workflow name is "Workflow - Test"? As of now it will be "Workflow_-_Test". Should we normalise such instances for a better workflow friendly name?